projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -100,12 +111,6 @@ Effects are injectable service classes with distinct parts:
 - Effects are subscribed to the `Store` observable.
 - Services are injected into effects to interact with external APIs and handle streams.
 
-<div class="alert is-helpful">
-
-**Note:** Since NgRx v15.2, classes are not required to create effects. Learn more about functional effects [here](#functional-effects).
-
-</div>
-
```

Comment:
why is this alert removed? we can keep the reference to functional effects

Reply:
This was my bad, thanks for adding it back @AdrianRomanski